mw-webapp/src/hooks/useChangeTheme.ts

```diff
@@ -0,0 +1,20 @@
+import {useState} from "react";
```

Comment:
Do we need this file? If yes? can you please explain why ) Let's discuss))

Reply:
I think we can try to get rid of useChangeTheme and useClickOutside files.  
This looks like an overwrapper

Reply:
remove useChangeTheme and useClickOutside files